packages/fxa-settings/src/components/Profile/index.stories.tsx

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Profile } from '.';
+import { MockedCache } from '../../models/_mocks';
+
+storiesOf('Components|Profile', module).add('default', () => (
+  <MockedCache>
+    <Profile />
```

Comment:
Since you've pulled latest, this is erroring - we have a `renderWithRouter` available for tests to give us the `LocationProvider` there but we need to modify it or make a mock that'll work in Storybook. This is what we need to fix the broken Storybooks in #6226 but I don't think I'm going to get to that today. I can address this over there if we don't have to view this for your PR.